docs/workflows.rst

```diff
@@ -405,57 +435,27 @@ mean-centered.
-The parameter estimates for each of the confounds will be retained, along with the residuals from
-the regression.
-The residuals from regressing the censored BOLD data on the censored confounds will be referred to
-as the ``denoised BOLD``.
-
-Additionally, the parameter estimates from each of the censored confounds will be used calculate
-residuals using the *full* (i.e., uncensored) confounds and BOLD data.
-The residuals from this step will be referred to as the ``uncensored, denoised BOLD``.
-The ``uncensored, denoised BOLD`` is later used for DCAN QC figures, but it is not written out to
-the output directory.
-
-
-Interpolation
--------------
-
-An interpolated version of the ``denoised BOLD`` is then created by filling in the high-motion
-outlier volumes with cubic spline interpolated data, as implemented in ``Nilearn``.
-
-.. warning::
-   In versions 0.4.0rc2 - 0.5.0, XCP-D used cubic spline interpolation,
-   followed by bandpass filtering.
-
-   However, cubic spline interpolation can introduce large spikes and drops in the signal
-   when the censored volumes are at the beginning or end of the run,
-   which are then propagated to the filtered data.
-
-   To address this, XCP-D now replaces interpolated volumes at the edges of the run with the
-   closest non-outlier volume's data, as of 0.5.1.
-
-The resulting ``interpolated, denoised BOLD`` is primarily used for bandpass filtering.
-
-
-Bandpass filtering [OPTIONAL]
------------------------------
+.. important::
 
-The ``interpolated, denoised BOLD`` is then bandpass filtered using a Butterworth filter.
-The resulting ``filtered, interpolated, denoised BOLD`` will only be written out to the output
-directory if the ``--dcan-qc`` flag is used, as users **should not** use interpolated data
-directly.
+   If you use the ``signal__`` approach, you must make sure that the signal regressors do not
+   reflect noise captured by the noise regressors.
+   Some ICA-based denoising methods, such as ME-ICA, can only distinguish specific types of noise,
+   so they may label other types of noise as "signal".
+   Since your confounds will be orthogonalized with respect to _all_ of your ``signal__``
+   regressors, this could lead to the imperfect removal of noise from your BOLD data.
 
-Bandpass filtering can be disabled with the ``--disable-bandpass-filter`` flag.
+The residuals from the second step are referred to as the ``denoised, interpolated BOLD``.
+The ``denoised, interpolated BOLD`` will only be written out to the output
+directory if the ``--dcan-qc`` flag is used,
+as users **should not** use interpolated data directly.
 
 
 Re-censoring
 ------------
 :class:`~xcp_d.interfaces.censoring.Censor`
 
 After bandpass filtering, high motion volumes are removed from the
-``filtered, interpolated, denoised BOLD`` once again, to produce ``filtered, denoised BOLD``.
+``denoised, interpolated BOLD`` once again, to produce ``denoised BOLD``.
```

Comment:
Maybe not for this PR, but I wonder which of these parts deserve special highlighting. maybe in a call-out box